libs/vscode/webview/src/lib/get-task-execution-schema.ts

```diff
@@ -170,21 +172,37 @@ export async function getTaskExecutionSchema(
 
 // Get information about where the user clicked if invoked through right click in the explorer context menu
 function getConfigValuesFromContextMenuUri(
+  schematic: TaskExecutionSchema,
   contextMenuUri: Uri | undefined,
   cliTaskProvider: CliTaskProvider
-): { path: string; directory: string; project?: string } | undefined {
+): { path: string; directory: string; project?: string, projectName?: string } | undefined {
   if (contextMenuUri) {
     const project = cliTaskProvider.projectForPath(contextMenuUri.fsPath);
     const projectName = (project && project.name) || undefined;
-    const path = contextMenuUri.fsPath
-      .replace(cliTaskProvider.getWorkspacePath(), '')
+
+    const workspacePath = cliTaskProvider.getWorkspacePath();
+    let path = contextMenuUri.fsPath
+      .replace(workspacePath, '')
       .replace(/\\/g, '/')
       .replace(/^\//, '');
 
+    const nxConfig = readAndCacheJsonFile('nx.json', workspacePath);
```

Comment:
I tried this with `workspaceLayout()` using a deep import from nx/workspace, but it wasn't giving me the configured path with a workspace I was testing on for some reason.